### DISCUSSION TITLE
Icon poll

### DISCUSSION THREAD

**Discussion by j20001970** [j20001970/GDMP#17] [Polls]
If the project will have a icon, what would you want it to look like?

| Icon A | Icon B | Icon C | Icon D |
|:----:|:----:|:----:|:----:|
|![iconA](https://github.com/j20001970/GDMP/assets/71766835/e431e985-1da8-4396-829d-cf1b9b05a2a7)|![iconB](https://github.com/j20001970/GDMP/assets/71766835/02666bb3-06f0-453d-970d-7ca314a3343a)|![iconC](https://github.com/j20001970/GDMP/assets/71766835/aa842eb8-4803-42e3-b52e-9f069d6e9183)|![iconD](https://github.com/j20001970/GDMP/assets/71766835/c185a891-d5f1-4599-a8da-0c09d86018f1)|

**Comment by GeorgeS2019** [comment]
Keep the poll open,  I need a few days to think

**Reply by GeorgeS2019** [reply]
@j20001970  Also consider how many users here actually use Godot3 and Godot4
